### ISSUE TITLE
UI: Don't use a toggle for yaml where the opposite means json

### ISSUE THREAD

**Issue by jsoref** [argoproj/argo-workflows#4970]
## Summary 

What happened/what you expected to happen?
On off toggles should be intuitive.

There's nothing intuitive that would hint that the opposite of yaml is json

https://workflows.apps.argoproj.io/event-sources/argo?sidePanel=true
![image](https://user-images.githubusercontent.com/2119212/106249734-bc05d700-61e0-11eb-98e3-49a35e865ea3.png)
![image](https://user-images.githubusercontent.com/2119212/106249754-c2944e80-61e0-11eb-9bea-87b62a49b749.png)

## Diagnostics

What Kubernetes provider are you using? 

What version of Argo Workflows are you running? 

```yaml
Paste a workflow that reproduces the bug, including status:
kubectl get wf -o yaml ${workflow} 
```

```
Paste the logs from the workflow controller:
kubectl logs -n argo $(kubectl get pods -l app=workflow-controller -n argo -o name) | grep ${workflow}
```

---

**Message from the maintainers**:

Impacted by this bug? Give it a 👍. We prioritise the issues with the most 👍.

**Comment by simster7** [comment]
Drop down? @alexec

**Comment by alexec** [comment]
Maybe change this to a button rather than a toggle? Either it says "JSON" or "YAML"?

**Comment by jsoref** [comment]
I do not like bots

**Comment by simster7** [comment]
@jsoref Would you like to submit a PR?

**Comment by jsoref** [comment]
I started playing w/ this, it seems like the right thing is to replace the button w/ either tabs or a widget (with spacing!) that presents two stateful buttons.

I think I may have just lost the work I was doing (I think I was drafting ideas in a github.com edit page instead of locally and I just restarted my browser...).

We use workflows, but mostly w/o interacting w/ the UI. So my bug filing was more to identify UX issues in the hopes that someone else would address them. That said, I might manage to make a PR, but waiting for me to fix all bugs I file is not a great strategy 😦 .
